docs/CustomJackServerName.md

```diff
@@ -0,0 +1,10 @@
+# Using custom JACK server name
+
+> ❕ This note is applicable only if you use JACK backend.
```

Comment:
```suggestion
!!! attention ""

    This note is applicable only if you use JACK backend.
```

Reply:
Actually this can be removed completely. Right now you only add one part of what Marcin suggested and it clearly states JACK in the heading. I don't see the benefit of adding this note.

Reply:
Okay, you're right, with JACK in the heading that's clear. Sorry for the noise!

Reply:
Though it totally makes sense for a full fledged page (that we need…).

Reply:
@ntonnaett If this PR introduces a page with temporary solution, should I just close it?)

Reply:
No. I'm very happy somebody's starting this. I don't want to risk not getting anything started just because we want more documentation. We'll always want more documentation. And documentation will never be finished.